build/targets/KoreBuild.SolutionBuild.targets

```diff
@@ -290,6 +290,8 @@ Runs the VSTest on all projects in the ProjectsToTest itemgroup.
 
   <PropertyGroup>
     <VSTestLogger Condition=" '$(VSTestLogger)' == '' AND '$(TEAMCITY_VERSION)' != '' ">trx</VSTestLogger>
+    <!-- ensure AppVeyor's auto test reporter remains enabled -->
+    <VSTestAutoReporters Condition="'$(APPVEYOR_API_URL)' != ''">true</VSTestAutoReporters>
```

Comment:
Then support `$(VSTestAutoReporters)` but get rid of the unnecessary special case for `'$(APPVEYOR_API_URL)' != ''` e.g.
``` xml
<VSTestAutoReporters Condition=" '$(VSTestAutoReporters)' == '' AND Condition=" '$(VSTestLogger)' != '' ">false</VSTestAutoReporters>
<VSTestAutoReporters Condition=" '$(VSTestAutoReporters)' == '' >true</VSTestAutoReporters>
```